opencensus/trace/samplers/__init__.py

```diff
@@ -12,11 +12,83 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from opencensus.trace.samplers.always_off import AlwaysOffSampler
-from opencensus.trace.samplers.always_on import AlwaysOnSampler
-from opencensus.trace.samplers.base import Sampler
-from opencensus.trace.samplers.probability import ProbabilitySampler
 
+DEFAULT_SAMPLING_RATE = 1e-4
 
-__all__ = ['Sampler', 'AlwaysOnSampler', 'AlwaysOffSampler',
-           'ProbabilitySampler']
+
+class Sampler(object):
+    """Base class for opencensus trace request samplers.
+
+    Subclasses must override :meth:`should_sample`.
+    """
+
+    def should_sample(self, span_context):
+        """Whether to sample this request.
+
+        :type span_context: :class:`opencensus.trace.span_context.SpanContext`
+        :param span_context: The span context.
+
+        :rtype: bool
+        :returns: Whether to sample the request according to the context.
+        """
+        raise NotImplementedError
+
+
+class AlwaysOnSampler(Sampler):
```

Comment:
Having `samplers.FoobarSampler` is much much better and more intuitive than `samplers.foobar_sampler.FoobarSampler`! Wish we could have that for propagators and other stuff.